### ISSUE TITLE
[2.0.0] Localization not working

### ISSUE THREAD

**Issue by NEZNAMY** [PaperMC/Velocity#474]
With b53 when entering an unknown command into console, it prints
```
[12:49:30] [main/INFO]: velocity.command.command-does-not-exist
```
Logs don't contain console input so I couldn't copy my input.
I did delete (rename) old velocity.toml from 1.1.x, so that shouldn't be the cause.

**Comment by astei** [comment]
I couldn't reproduce this, but my system is set to English (United States) language, so that probably is why it's working for me. I'm going to take a closer look.

**Comment by NEZNAMY** [comment]
I could try some debug jar if that helps.
Edit: Just checked my system settings and I'm also using english US.

**Comment by astei** [comment]
I believe I've fixed the issue. Tested with a fake locale we have (en_UD) and a locale we don't have yet (es_ES), all is good now. Go ahead and try on your end.

**Comment by NEZNAMY** [comment]
Still broken in b54
```
[13:45:04] [main/WARN]: Velocity 2.0.0 is under active development and WILL NOT WORK WITH YOUR EXISTING PLUGINS!
[13:45:05] [main/INFO]: Booting up Velocity 2.0.0-SNAPSHOT (git-efa9f6f3-b54)...
[13:45:05] [main/INFO]: Connections will use NIO channels, Java compression, Java ciphers
[13:45:05] [main/WARN]: Player info forwarding is disabled! All players will appear to be connecting from the proxy and will have offline-mode UUIDs.
[13:45:05] [main/INFO]: Loading plugins...
[13:45:05] [main/INFO]: Loaded plugin test 1.0.0 by me
[13:45:05] [main/INFO]: Loaded 1 plugins
[13:45:05] [Netty NIO Boss #0/INFO]: Listening on /[0:0:0:0:0:0:0:0]:25577
[13:45:05] [main/INFO]: Done (1.5s)!
[13:45:06] [main/INFO]: velocity.command.command-does-not-exist
```
Perhaps also log console commands?

**Comment by astei** [comment]
I'm assuming you're using Windows. Well, I guess I should not be surprised because it is working on my Linux system (Fedora Workstation 33). I guess it's time to break out the virtual machine...

**Comment by astei** [comment]
Confirmed that this is a Windows issue.

**Comment by astei** [comment]
Narrowed it down to https://stackoverflow.com/questions/9873845/java-7-zip-file-system-provider-doesnt-seem-to-accept-spaces-in-uri, so it's not a Windows issue.

**Comment by NEZNAMY** [comment]
Well they say it was fixed in Java 8? https://stackoverflow.com/a/14034572

**Comment by astei** [comment]
Okay, my initial assessment was wrong (as I later found out). The issue boils down mostly to it not realizing it's running from a JAR for some reason.

**Comment by astei** [comment]
Should be fixed by 688babb, thanks!
